index.html

```diff
@@ -918,7 +953,7 @@ <h4>publicKeyMultibase</h4>
                 <a href="https://w3id.org/security#publicKeyMultibase">security-vocab</a>
               </td>
               <td>
-                <a href="https://www.w3.org/ns/did/v1">DID Core</a>
+                <a href="https://www.w3id.org/security/v3-unstable">https://www.w3id.org/security/v3-unstable</a>
```

Comment:
I've noticed in some cases we use www and other cases we don't in context URLs. What's the way you discern when it should or shouldn't be used?

Reply:
For JSON-LD Context URLs... whatever URL is shorter and doesn't lead to more redirects than necessary is what I tend to use.

Reply:
That is both helpful (it draws a fuzzy line) and non-helpful (I have to go look into the number of redirects) what's interesting about this is that we've had issues with processing did documents because we were doing static analysis on the context urls included and in some cases the URLs were valid and resolvable but the www was throwing things off. In that case, the problem and solution lies within our system effectively but my intent with trying to be simple and consistent here was to effectively make it easier for others to avoid the issues we ran into in strongly typed systems.

Proposal: For this registry, when using w3.org domain use www subdomain and when using w3id.org domain do not include www. That seems to align with the way we've been using context URLs throughout implemenations without breaking spec compliance (VC and DID spec use www in the URLs that they normatively define).